app/models/vaccine.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+# Represents a vaccination for a Patient
+class Vaccine < ApplicationRecord
+  belongs_to :patient
+
+  VACCINE_STANDARDS = Rails.configuration.vaccine_standards.freeze
+  MAX_DOSE_NUMBER = 2
```

Comment:
should this number be defined in the config based on the `group_name` and `product_name`? since the J&J vaccine only requires 1 dose and future vaccines might require more/less doses?

Reply:
Given you and @holmesie both brought up moving this to the config, I went ahead and added a "num_doses" for each vaccine in the config and calculated `MAX_DOSE_NUMBER` based on it: https://github.com/SaraAlert/SaraAlert/pull/721/commits/66e9a6d482854054a97a2f0ac8c1ec4c0c6af3ca#diff-69f2cba2092662388d22d16e62c30957e4c13b866314226ed2ad2580ccf764b6R10

Reply:
thanks Kylie, would it make sense to only allow the `MAX_DOSE_NUMBER` based on the `group_name` and `product_name` that way a user can't enter in dose_number 2 for J&J or would that just make things too complicated?

Reply:
Yeah when we were discussing all this we considered a few options for further validation (checking administration dates base on dose number, verifying you don't have records for the same vaccine with the same dose number, etc) but we ultimately landed on not trying to make Sara Alert a vaccine management system and keeping it simple for now, at least for this first iteration

Reply:
sounds good!